### PR TITLE
Feature verbose load params

### PR DESCRIPTION
**Pull request by BenjaminBossan**
When verbose, load_params now alerts the user about whether parameters could or could not be loaded successfully into the current net.

### CODE REVIEW

File: nolearn/lasagne/tests/test_base.py [hunk 1]
```diff
@@ -114,6 +114,22 @@ def test_save_params_to_path(self, net, net_fitted, X_test, y_pred):
         net_loaded.load_params_from(path)
         assert np.array_equal(net_loaded.predict(X_test), y_pred)
 
+    def test_load_params_from_message(self, net, net_fitted, X_test, y_pred,
```
Comment (dnouri) [inline nolearn/lasagne/tests/test_base.py]: `net_fitted` not needed here?

File: nolearn/lasagne/tests/test_base.py [hunk 2]
```diff
@@ -114,6 +114,22 @@ def test_save_params_to_path(self, net, net_fitted, X_test, y_pred):
...
+                                      capsys):
+        net_loaded = clone(net)
```
Comment (dnouri) [inline nolearn/lasagne/tests/test_base.py]: I think I used `net_loaded` in tests for nets that were loaded from disk.  Maybe `net` alone is fine.

File: nolearn/lasagne/tests/test_base.py [hunk 3]
```diff
@@ -114,6 +114,22 @@ def test_save_params_to_path(self, net, net_fitted, X_test, y_pred):
...
+        net_loaded.verbose = 1
+        net_loaded.load_params_from(net_fitted)
+
+        out = capsys.readouterr()[0]
+        message = """Loaded parameters to layer 'hidden1' (shape 784x512).
+Loaded parameters to layer 'hidden1' (shape 512).
+Loaded parameters to layer 'hidden2' (shape 512x512).
+Loaded parameters to layer 'hidden2' (shape 512).
+Loaded parameters to layer 'output' (shape 512x10).
+Loaded parameters to layer 'output' (shape 10).
```
Comment (dnouri) [inline nolearn/lasagne/tests/test_base.py]: I'm confused, why is `output` listed twice?

File: nolearn/lasagne/handlers.py [hunk 4]
```diff
@@ -24,7 +24,7 @@ def table(self, nn, train_history):
         info_tabulate = OrderedDict([
             ('epoch', info['epoch']),
             ('train loss', "{}{:.5f}{}".format(
-                ansi.BLUE if info['train_loss_best'] else "",
+                ansi.CYAN if info['train_loss_best'] else "",
```
Comment (dnouri) [inline nolearn/lasagne/handlers.py]: Looks like this contains commits from your other branch.

File: nolearn/lasagne/base.py [hunk 5]
```diff
@@ -447,12 +447,27 @@ def load_params_from(self, source):
...
+
         for key, values in source.items():
             layer = self.layers_.get(key)
             if layer is not None:
                 for p1, p2v in zip(layer.get_params(), values):
-                    if p1.get_value().shape == p2v.shape:
+                    shape1 = p1.get_value().shape
+                    shape2 = p2v.shape
+                    shape1s = 'x'.join(list(map(str, shape1)))
+                    shape2s = 'x'.join(list(map(str, shape2)))
```
Comment (dnouri) [inline nolearn/lasagne/base.py]: The `list` is unnecessary here.  `''.join` can deal with any iterable.

File: nolearn/lasagne/tests/test_base.py (hunk 3 above)
Comment (BenjaminBossan) [inline nolearn/lasagne/tests/test_base.py]: Each layer appears twice, once for weights and once for bias. You can tell from the shape.

File: nolearn/lasagne/handlers.py (hunk 4 above)
Comment (BenjaminBossan) [inline nolearn/lasagne/handlers.py]: I reset the commit and rebased with current master. I don't know why it's still there :(

**Pull request merged**